packages/eslint-config-eslint/default.yml

```diff
@@ -92,6 +99,7 @@ rules:
     space-unary-ops: ["error", {words: true, nonwords: false}]
     spaced-comment: ["error", "always", { exceptions: ["-"]}]
     strict: ["error", "global"]
+    template-curly-spacing: ["error", "never"]
     valid-jsdoc: ["error", {
```

Comment:
This is the only one I'm unsure of. I think I'd prefer "always", but happy to go along with whatever is the consensus.